src/openforms/js/components/admin/form_design/registrations/index.js

```diff
@@ -49,4 +49,12 @@ export const BACKEND_OPTIONS_FORMS = {
     form: ZGWOptionsForm,
     onStepEdit: onZGWStepEdit,
   },
+  'stuf-zds-create-zaak:ext-utrecht': {
```

Comment:
this part will require backporting - can you create a PR for it to `stable/2.7.x`?